includes/Core/Authentication/Authentication.php

```diff
@@ -314,7 +314,7 @@ public function get_disconnect_url() {
 			'splash',
 			array(
 				'googlesitekit_disconnect' => 1,
-				'nonce'                    => wp_create_nonce( 'disconnect' ),
+				'nonce'                    => wp_generate_password(),
```

Comment:
This needs to remain as a WP nonce or it will fail verification
https://github.com/google/site-kit-wp/blob/35eef8d555bd8abe887244ae7689385dbd3ad48c/includes/Core/Authentication/Authentication.php#L359-L363